### ISSUE TITLE
Repetitive and non-informative editor's label

### ISSUE THREAD

**Issue by Comandeer** [ckeditor/ckeditor5#11836]
## 📝 Provide detailed reproduction steps (if any)

1.  Open https://ckeditor.com/docs/ckeditor5/latest/examples/builds/classic-editor.html
2.  Launch VoiceOver (or some other screen reader).
3.  Focus the editor.

### ✔️ Expected result

The editor is announced in a concise and non-redundant way.

### ❌ Actual result

The editor is announced as "Rich Text Editor, \<root name>, edit text. \<content and selection info>. Rich Text Editor, application.

### ❓ Possible solution

Both labels – for the application region and editable ones – should be different and easily differentiated. See [CKE4's implementation](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_a11y.html#announcing-the-editor-on-the-page) as an example.

## 📃 Other details

*   Browser: all
*   OS: all
*   Installed CKEditor plugins: N/A

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by mlewand** [comment]
In NVDA or JAWS the readout is even worse as it goes

"Rich text editor, Rich text editor (...)"

Reason for this is that both textbox and application element have the same label.

![](https://user-images.githubusercontent.com/5353898/171403445-d0be32f6-3dad-46b0-afaf-4a92efca1ce8.png)

The solution for now is to remove "Rich Text Editor" prefix from **textbox's** label (as it is inherited from role=application element).
